src/wmtk/simplex/internal/SimplexLessFunctor.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <wmtk/Mesh.hpp>
+#include <wmtk/simplex/Simplex.hpp>
+
+namespace wmtk::simplex::internal {
+struct SimplexLessFunctor
+{
+    const Mesh& m;
+
+    SimplexLessFunctor(const Mesh& mm)
+        : m{mm}
+    {}
+
+    bool operator()(const Simplex& s0, const Simplex& s1) const
+    {
+        return m.simplex_is_less(s0, s1);
```

Comment:
heh, i was teh one who asked for the change, but it's sorta ugly that this makes sense to be simplex_is_less and the other one is simplices_are_equal . In the end I think that the equal/less belong in a `struct SimplicialComplex { const Mesh& m_mesh};` datastructure. Then you'd just have `less_than` and `equal` without having to clarify simplex (+ we could remove Simplex from the Mesh class)

Reply:
You mentioned that before and I have not fully understood your point here so far. For simplex comparison, we need functors in some way. I don't see how this can be done within one class for both, `less_than` and `equal`. I guess I am missing some knowledge here.